src/runtime/config/configuration-qemu-sev-snp.toml.in

```diff
@@ -0,0 +1,664 @@
+# Copyright 2022 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# XXX: WARNING: this file is auto-generated.
+# XXX:
+# XXX: Source file: "@CONFIG_QEMU_IN@"
+# XXX: Project:
+# XXX:   Name: @PROJECT_NAME@
+# XXX:   Type: @PROJECT_TYPE@
+
+# Basing SNP updates on this post from Joana
+# https://github.com/kata-containers/kata-containers/blob/main/docs/how-to/how-to-run-kata-containers-with-SNP-VMs.md
+
+[hypervisor.qemu]
+path = "@QEMUPATH@"
```

Comment:
```suggestion
path = "@QEMUSNPPATH@"
```
if you align with https://github.com/kata-containers/kata-containers/pull/6062#issuecomment-1406581382